server/graphql/general/src/queries/log.rs

```diff
@@ -0,0 +1,50 @@
+use async_graphql::{Context, *};
+use graphql_core::{standard_graphql_error::validate_auth, ContextExt};
+use service::auth::{Resource, ResourceAccessRequest};
+
+#[derive(SimpleObject)]
+pub struct LogNode {
+    pub file_names: Option<Vec<String>>,
+    pub file_content: Option<Vec<String>>,
+}
+
+impl LogNode {
+    fn from_domain(file_names: Option<Vec<String>>, file_content: Option<Vec<String>>) -> LogNode {
+        LogNode {
+            file_names,
+            file_content,
+        }
+    }
+}
+
+pub fn log_file_names(ctx: &Context<'_>) -> Result<LogNode> {
+    validate_auth(
+        ctx,
+        &ResourceAccessRequest {
+            resource: Resource::QueryLog,
+            store_id: None,
+        },
+    )?;
+
+    let service_provider = ctx.service_provider();
+    let log_service = &service_provider.log_service;
+    let file_names = log_service.get_log_file_names()?;
+
+    Ok(LogNode::from_domain(Some(file_names), None))
+}
+
+pub fn log_content(ctx: &Context<'_>, file_name: Option<String>) -> Result<LogNode> {
```

Comment:
can we also have a query to get the current log level? 🙏 
I'm thinking that for the UI we'd want to select the current state

Reply:
[a5856a5](https://github.com/openmsupply/open-msupply/pull/1897/commits/a5856a503ca45d49a9c89d63654f294c5e3d9c38)